### PR TITLE
feat: Add HowTo: Managing invoices

### PR DESCRIPTION
**Pull request by colder-is-better**
We show how to view paid and outstanding invoices, how to view, download or pay oustanding invoices, and how to change the email address for receiving invoices.

### CODE REVIEW

File: docs/howto/account-billing/manage-invoices.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+![Payment options](assets/mgmnt-inv/shot-04.png)
+
+Finally, you may change the email address for receiving invoices. At 
+the right-hand side of the dashboard, click the user icon at the top of 
+the page. A pane titled _Account settings_ will slide over. There are 
+three tabs in that pane; _Customer Info_, _Contact_, and _Settings_. 
+Click the middle one, _Contact_. If there is only one email address, it 
+will have all available roles activated --- including the _Billing_ 
+role.
```
Comment (fghaas) [inline docs/howto/account-billing/manage-invoices.md:44]:
```suggestion
## Changing your invoice email address

Finally, you may change the email address for receiving invoices. At 
the right-hand side of the dashboard, click the user icon at the top of 
the page. A pane titled _Account settings_ will slide over. 

There are three tabs in that pane:
* _Customer Info_,
* _Contact_, 
* _Settings_. 

Click the middle one, _Contact_. If there is only one email address, it 
will have all available roles activated --- including the _Billing_ 
role.
```

File: docs/howto/account-billing/manage-invoices.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+You may quickly review your invoices regarding the monthly usage of 
+{{brand}} resources, selectively pay outstanding invoices, save any 
+number in PDF format, and change the email address for receiving such 
+invoices.
+
+First, fire up your favorite web browser, navigate to the 
+[{{gui}}](https://{{gui_domain}}) start page, and log into your 
+{{brand}} account.
+
+Make sure the vertical pane on the left-hand side of the dashboard is
```
Comment (fghaas) [inline docs/howto/account-billing/manage-invoices.md:12]:
```suggestion
## Listing invoices

Make sure the vertical pane on the left-hand side of the dashboard is 
```

File: docs/howto/account-billing/manage-invoices.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+![All invoices](assets/mgmnt-inv/shot-01.png)
+
+Any paid invoice has a green circle-with-tick icon at the left of its 
+row. Also, look at the _Outstanding Amount_ column, where there is a 
+value of `0.00` for each paid invoice.
+
+![Paid invoices](assets/mgmnt-inv/shot-02.png)
+
+Notice the three-dot orange icon at the right or any invoice row. By
```
Comment (fghaas) [inline docs/howto/account-billing/manage-invoices.md:25]:
```suggestion
## Displaying invoice details

Notice the three-dot orange icon at the right or any invoice row. By 
```

File: docs/howto/account-billing/manage-invoices.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+![Paid invoices](assets/mgmnt-inv/shot-02.png)
+
+Notice the three-dot orange icon at the right or any invoice row. By 
+clicking on it, you get a pop-up with specific options regarding the 
+corresponding invoice. For example, you can view the invoice's PDF or 
+download it onto your computer.
+
+![Options regarding selected invoice](assets/mgmnt-inv/shot-03.png)
+
+There is also the _Pay this invoice_ option. Choose it for an
```
Comment (fghaas) [inline docs/howto/account-billing/manage-invoices.md:32]:
```suggestion
## Paying outstanding invoices

There is also the _Pay this invoice_ option. Choose it for an 
```

**Review by fghaas — COMMENTED** [review]
Looks good in general! I have just a few suggestions.

File: docs/howto/account-billing/manage-invoices.md (hunk 1 above)
Comment (colder-is-better) [reply docs/howto/account-billing/manage-invoices.md]: Section titles, yes :)

**Comment by colder-is-better** [comment]
Nope, no objections whatsoever :)

**Pull request merged**